src/EVM/Format.hs

```diff
@@ -495,6 +497,13 @@ formatPartialShort = \case
 formatSomeExpr :: SomeExpr -> Text
 formatSomeExpr (SomeExpr e) = formatExpr $ Expr.simplify e
 
+formatState :: Map.Map (Expr EAddr) (Expr EContract) -> Text
+formatState store = indent 2 $ T.unlines (fmap (\(k,v) ->
+              T.unlines
+                [ formatExpr k <> ":"
+                , indent 2 $ formatExpr v
+                ]) (Map.toList store))
+
```

Comment:
I reuse this `formatState` later in SymExec for equivalence checking. Allows me to print the state when it differs between two contracts.